CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+---
+
+## [6.0.2] - 2023-03-24
+
+### Changed
+
+- Avoid logging normal exits ([#140](https://github.com/primait/amqpx/pull/140))
+
+---
+
+## [6.0.1] - 2023-03-17
+
+### Added
+
+- [[STAFF-31](https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/STAFF-31)] Add proper shutdown handling to Amqpx generic producers and consumers ([#138](https://github.com/primait/amqpx/pull/138))
+- Add retry mechanism for publish ([#121](https://github.com/primait/amqpx/pull/121))
```

Comment:
I think i would keep integrity in the file putting the related PR link at the start or at the end of the statement for each link

Reply:
I opted to remove links to youtrack cards and have only issue link, wdyt?